mirrord/protocol/src/tcp.rs

```diff
@@ -113,7 +113,7 @@ pub enum StealType {
     /// PROTOCOL
     FilteredHttp(Port, Filter),
     /// Steal HTTP traffic matching a given filter - supporting more than once kind of filter
-    FilteredHttpMany(Port, HTTPFilter),
+    FilteredHttpEx(Port, HttpFilter),
```

Comment:
I don't get the name 😬

Reply:
too much time on windows but this means `Extended`